src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -154,7 +154,6 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     path(['metadata', 'options', 'allowAdvertising'], pageData),
     adsEnabled,
     showAdsBasedOnLocation,
-    process.env.SIMORGH_APP_ENV !== 'live',
```

Comment:
I'm pretty sure the tests for `StoryPage` will now fail as we have tests that check ads only work test but not live?

Reply:
What tests? They all passed locally

Reply:
These tests all set the environment to be test: https://github.com/bbc/simorgh/blob/enable-sty-ads-on-live/src/app/pages/StoryPage/index.test.jsx#L421..L567 

I think that is no longer significant as the environment doesn't matter now is ads are enabled on all environments, I think the references to environments should be removed from these tests: https://github.com/bbc/simorgh/blob/enable-sty-ads-on-live/src/app/pages/StoryPage/index.test.jsx#L276..L567 

As it happens we never actually had a test to cover ads should not show on live like I thought we did 🙈